e2e/volume_ops_test.go

```diff
@@ -405,6 +405,12 @@ func TestVolumeOptions(t *testing.T) {
 	settableKey := "afr.use-compound-fops"
 	optionReq.Options = map[string]string{settableKey: "on"}
 	r.Nil(client.VolumeSet(volname, optionReq))
+
+	var resetOptionReq api.VolOptionResetReq
+	resetOptionReq.Options = append(resetOptionReq.Options, "afr.use-compound-fops")
```

Comment:
cover multiple volume options in e2e test cases.
it will help to identify issues more quickly.